### DISCUSSION TITLE
Why "Capture log call" is displaying details

### DISCUSSION THREAD

**Discussion by VeggieMonster** [pytest-dev/pytest-html#520] [Q&A]
Why "Capture log call" is displaying details about the code line where the logging occours?
Is there any setting available to hide it?

![Unbenannt](https://user-images.githubusercontent.com/100716707/176649914-bb7606b5-25cd-4022-9ef0-f364e5c25b0d.JPG)

**Comment by RonnyPfannschmidt** [comment]
its not quite clear what you are asking for - https://docs.pytest.org/en/7.1.x/how-to/logging.html has the configuration examples

**Reply by VeggieMonster** [reply]
Hello,

so I have configured the pytest.ini that in CLI or in the log file the output is displayed with timestamps and without the additional message "root:logging.py:55"

![llc](https://user-images.githubusercontent.com/100716707/177551080-3a713ccb-e5de-45b7-b522-37dcccd2037e.JPG)

It will be fine to have also the timestamps in the html report and ommit the entry of "root:logging.py:55"

![lhtml](https://user-images.githubusercontent.com/100716707/177551118-6b121b80-572e-4602-be72-cf02655a4565.JPG)

**Comment by BeyondEvil** [comment]
Remind me about this when we've released v4.0.0
